src/backend/distributed/commands/sequence.c

```diff
@@ -177,6 +177,14 @@ ExtractColumnsOwningSequences(Oid relationId, List **columnNameList,
 			continue;
 		}
 
+#if PG_VERSION_NUM >= PG_VERSION_12
```

Comment:
I think the name of the function `ExtractColumnsOwningSequences` is pretty confusing. Maybe `ExtractDefaultColumnsAndOwningSequences()` and update the function comment a little?

Also, even `DropAndMoveDefaultSequenceOwnerships` -> `DropDefaultExpressiosAndMoveOwningSequnenceOwnerships` or such

Reply:
makes sense, I will address those

Reply:
maybe ExtractColumnsOwningSequences -> ExtractDefaultColumnsAnd**Owned**Sequences

Reply:
similarly, maybe DropAndMoveDefaultSequenceOwnerships -> DropAndMoveDefaultSequenceOwnerships DropDefaultExpressionsAndMove**Owned**SequenceOwnerships